documentation/obtain-storm/apple-silicon.md

```diff
@@ -5,11 +5,23 @@ layout: default
 
 <h1>Building for ARM-based Apple Silicon Systems </h1>
 
-{:.alert .alert-info}
-_Native_ code compilation on ARM-based Apple Silicon systems is not yet supported. However, compilation with x86 emulation (Rosetta 2) does work just fine. We provide detailed installation steps for that below.
-Native ARM compilation is work in progress.
+## Native Compilation to an ARM binary (recommended)
+
+Storm compiles natively on ARM-based apple silicon processors. 
+However, there are a few points to consider for troubleshooting.
+
+* Make sure you have the latest version of Storm (April 2023 or later)
```

Comment:
Better already reference the upcoming release? "... version of Storm (Version 1.8.0 or later)"